docs/platforms/javascript/common/configuration/integrations/featureflags.mdx

```diff
@@ -25,46 +25,24 @@ notSupported:
 
 <Alert level="info">
 
-This integration only works inside a browser environment. If you are using a specific
-feature flag provider, refer to the supported list [here](/platforms/javascript/feature-flags/#enable-evaluation-tracking) before setting up this integration.
+This integration only works inside a browser environment, and you must install
+from npm.
 
 </Alert>
 
-The Feature Flags integration allows you to manually track feature flag evaluations through an API. These evaluations are held in memory and sent to Sentry when an error occurs.
-**At the moment, we only support boolean flag evaluations.**
-
-_Import names: `Sentry.featureFlagsIntegration` and `type Sentry.FeatureFlagsIntegration`_
-
-## Install
-
-Install your platform's Sentry SDK from npm.
+<Alert level ="info">
 
-## Configure
+If you are using an external feature flag provider, refer to the supported list [here](/platforms/javascript/feature-flags/#enable-evaluation-tracking) before setting up this integration.
```

Comment:
Generally, try to avoid "here" links -- you can link from more contextual text

```suggestion
If you are using an external feature flag provider, refer to the [supported list](/platforms/javascript/feature-flags/#enable-evaluation-tracking) before setting up this integration.
```